gandalf.js

```diff
@@ -9,34 +9,30 @@
  * Copyright 2016 Henrik Ingo (@henrikingo)
```

Comment:
SCRIPT-You did a great job handling all the possible error states in the UI. It enhances the user experience significantly.